### PR TITLE
add NEWS entry for change in default color for info messages [#18442]

### PR DESCRIPTION
**Pull request by KristofferC**
cc @eschnett

### CODE REVIEW

**Comment by eschnett** [comment]
You can also add a link to <http://docs.julialang.org/en/release-0.5/manual/interacting-with-julia/#customizing-colors>. This gives further information, e.g. also about how to change the warning colour (which also changed).

**Comment by KristofferC** [comment]
That change is still in a PR though.

**Comment by KristofferC** [comment]
Updated with link to manual. Should be good to go.

File: NEWS.md [hunk 1]
```diff
@@ -17,6 +17,7 @@ Language changes
   * The default color for info messages has been changed from blue to cyan.
     This can be changed back to the original color by setting the environment variable `JULIA_INFO_COLOR` to `"blue"`.
     One way of doing this is by adding `ENV["JULIA_INFO_COLOR"] = :blue` to the `.juliarc.jl` file.
+    For more information regarding customizing colors in the REPL, see this [manual section]( http://docs.julialang.org/en/release-0.5/manual/interacting-with-julia/#customizing-colors).
```
Comment (tkelman) [inline NEWS.md]: latest not release-0.5?

**Review by kshyatt — APPROVED** [review]

**Comment by KristofferC** [comment]
"manual section" -> "section of the manual"?

**Comment by KristofferC** [comment]
Googling a bit it seems "manual section" is fine.

**Pull request merged**